src/pages/product-list/components/display/display.tsx

```diff
@@ -55,7 +56,29 @@ const Display = (props: IProps) => {
           </Box>
         </Box>
       ) : (
-        <>Loading data..</>
+        <Box p={3}>
```

Comment:
why we have the same code twice?

Reply:
Because I don't have Docker installed still, so I made that in case we don't have data from BE (because I don't have Docker), it renders the static content so I can work in that section

Reply:
Yes but we shouldn't commit that. If you use that only for yourself then do not commit it. We should have a spinner there instead of loaidng data text